recipe/0004-Use-central-cuda-capabilities.patch

```diff
@@ -1,13 +1,22 @@
+From 674b1e1eafd3ddbcabbd990d7da3c82eb539d851 Mon Sep 17 00:00:00 2001
+From: Deepali Chourasia <deepch23@in.ibm.com>
+Date: Tue, 8 Nov 2022 07:44:57 +0000
+Subject: [PATCH] use centralized cuda capabilities
+
+---
+ cmake/Utils.cmake | 10 ++++++++--
+ 1 file changed, 8 insertions(+), 2 deletions(-)
+
 diff --git a/cmake/Utils.cmake b/cmake/Utils.cmake
-index 963c494c..64bbfdd7 100644
+index f28c1f27..f0b6f1b2 100644
 --- a/cmake/Utils.cmake
 +++ b/cmake/Utils.cmake
 @@ -90,8 +90,13 @@ function(format_gencode_flags flags out)
    endif()
    # Set up architecture flags
    if(NOT flags)
 -    if (CUDA_VERSION VERSION_GREATER_EQUAL "11.1")
--      set(flags "52;60;61;70;75;80;86")
+-      set(flags "50;60;70;80")
 +    if (CUDA_VERSION VERSION_GREATER_EQUAL "11.4")
 +      string(REPLACE "," ";" tmpflags "$ENV{cuda_levels}")
```

Comment:
Is `$ENV{cuda_levels}` a subset of `"52;60;61;70;75;80;86"`?

Reply:
We have `cuda_levels` defined at https://github.com/open-ce/open-ce/blob/main/envs/conda_build_config.yaml#L394-L398 and this change to pick up the values configured there.

Reply:
Hmm... Ours are `3.7,5.2,6.0,7.0,7.5,8.0,8.6 ` , correct? I would propose to remove `3.7` - if XGBoost no longer supports it, I'm ok with dropping it from openCE as well..